pkg/storegateway/stats.go

```diff
@@ -75,6 +75,7 @@ type queryStats struct {
 	streamingSeriesEncodeResponseDuration       time.Duration
 	streamingSeriesSendResponseDuration         time.Duration
 	streamingSeriesOtherDuration                time.Duration
+	streamingSeriesIndexLoadDuration            time.Duration
```

Comment:
I would clarify it here as well.

```suggestion
	streamingSeriesIndexHeaderLoadDuration            time.Duration
```